CMakeLists.txt

```diff
@@ -28,6 +28,12 @@ target_link_libraries(project1 project1_lib)
 # Testing
 ################################
 if (test)
+  # only for apple computer
```

Comment:
Gotcha, thanks! Looks good. Can you just remove the "# only for apple computer" line because that adds no meaning - this is easily understood by your if-statement. Thanks.